### PR TITLE
Update the index of docker command line

### PR DESCRIPTION
**Pull request by yuexiao-wang**
**- What I did**
Update the index of docker command line

**- How I did it**

**- How to verify it**

**- Description for the changelog**

Signed-off-by: yuexiao-wang <wang.yuexiao@zte.com.cn>

### CODE REVIEW

File: docs/reference/commandline/index.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ read the [`dockerd`](dockerd.md) reference page.
 | [rmi](rmi.md) | Remove one or more images                                    |
 | [save](save.md) | Save images to a tar archive                               |
 | [tag](tag.md) | Tag an image into a repository                               |
+| [image prune](image_prune.md) | Remove unused images                         |
```
Comment (thaJeztah) [inline docs/reference/commandline/index.md]: perhaps we should try to keep this in alphabetical order (although the new `docker image load`, etc. , subcommands haven't been added to the docs yet.

Can you move `image prune` above `rmi` ? (same for the other changes below

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: docs/reference/commandline/index.md (hunk 1 above)
Comment (yuexiao-wang) [inline docs/reference/commandline/index.md]: Go it

**Comment by yuexiao-wang** [comment]
@thaJeztah 
It has been done

**Review by thaJeztah — APPROVED** [review]
LGTM

**Review by vdemeester — APPROVED** [review]
LGTM 🐸

**Pull request merged**